eclipsePlugin/build.gradle

```diff
@@ -74,6 +74,7 @@ task copyLibsForEclipse (type: Copy) {
     into "lib"
     include "*.jar"
     exclude "*xml-apis*.jar"
+    exclude "jdepend*.jar"
```

Comment:
Why is this still necessary? Can’t we switch `configurations.embeddedLibs` to the *compile* rather than *test* scope of `project(':spotbugs')`?

@jsotuyod This may need a Gradle expert. (As a Maven guy myself I am continuously confused by the script-ness of Gradle…)